chromium_src/third_party/blink/renderer/modules/storage/storage_controller.h

```diff
@@ -0,0 +1,18 @@
+/* Copyright (c) 2020 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#ifndef BRAVE_CHROMIUM_SRC_THIRD_PARTY_BLINK_RENDERER_MODULES_STORAGE_STORAGE_CONTROLLER_H_
+#define BRAVE_CHROMIUM_SRC_THIRD_PARTY_BLINK_RENDERER_MODULES_STORAGE_STORAGE_CONTROLLER_H_
+
+#define BRAVE_STORAGE_CONTROLLER_H                         \
+ public:                                                   \
+  static bool CanAccessStorageAreaWithoutEphemeralStorage( \
```

Comment:
why is this a static method on StorageController? I can't see any reason for it to be part of StorageController and seems like it would work just as well as an anonymous method added to dom_window_storage.cc

Reply:
Hmm, IIUC, `CanAccessStorageAreaWithoutEphemeralStorage` replaces the original `CanAccessStorageArea` in storage_controller.h. And the current `CanAccessStorageArea` returns true if cookie allowed or ephemeral flag enabled.
It seems we can't  move this to dom_window_storage.cc, for the `StorageArea::CanAccessStorage()` needs it.

Reply:
then it can just be a regular method and not a static method on a class that requires a patch

Reply:
OK, this change has been removed in the new patch.